case-lib/lib.sh

```diff
@@ -397,7 +399,27 @@ is_zephyr()
 
 logger_disabled()
 {
-    [[ ${OPT_VAL['s']} -eq 0 ]]
+    local ldcFile
+    # Some firmware/OS configurations do not support logging.
+    ldcFile=$(find_ldc_file) || {
+        dlogi '.ldc dictionary file not found, SOF logs collection disabled'
+        return 0 # 0 is 'true'
+    }
+
+    # Disable logging when available...
+    if [ ${OPT_VAL['s']} -eq 0 ]; then
+        return 0
+    fi
+
+    # ... across all tests at once.
+    # In the future we should support SOF_LOGGING=etrace (only), see
```

Comment:
Need more clarification. In future we will support only etrace? why? slogger seems useful to me

Reply:
See #726 referenced on the next line.